env.d/development/common

```diff
@@ -15,7 +15,7 @@ MAGNIFY_API_URL="http://localhost:8070/api"
 KEYCLOAK_URL="http://localhost:8080"
 
 # Jitsi jwt settings
-JITSI_URL="https://meet.jit.si"
+JITSI_URL="meeting.education"
```

Comment:
@sampaccoud

Reply:
@PanchoutNathan do you really need a hostname or you're making a url of it in the frontend? It would be better to be able to configure the protocol as well?

Reply:
The jitsi SDK need a hostname :)